packages/jest-cli/src/__tests__/generateEmptyCoverage.test.js

```diff
@@ -35,14 +38,34 @@ module.exports = {
 };`;
 
 it('generates an empty coverage object for a file without running it', () => {
+  const coverageMap = istanbulCoverage.createCoverageMap({});
+  const sourceMapStore = libSourceMaps.createSourceMapStore();
+  const rootDir = '/tmp';
+  const filepath = path.join(rootDir, './sum.js');
```

Comment:
`filepath` must be under `rootDir`. Otherwise it [will not be instrumented](https://github.com/facebook/jest/blob/master/packages/babel-jest/src/index.js#L138).